samples/RhinoCompute/AuthToken.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Rhino.Compute
+{
+    public static class AuthToken
+    {
+        public static string Get ()
+        {
+            // TODO: Add authorization token here
+            return "<Add authorization token here>";
```

Comment:
Nice idea! Maybe we could extend this in future to read from a text file or environment variable?

Reply:
Yes, a text file would be a good idea. Or maybe a simple dialog which stores it in a text file so you can provide the token only once?